### ISSUE TITLE
Bug: autofix of `attribute-order` removes empty string attribute/argument values

### ISSUE THREAD

**Issue by IgnaceMaes** [ember-template-lint/ember-template-lint#3054]
## Description

The auto fix of `attribute-order` removes empty string values for attributes and arguments.

## Example

Before:

```hbs
<SomeComponent @b="" @a="foo" />
```

After:

```hbs
<SomeComponent @a="foo" @b />
```

Expected:

```hbs
<SomeComponent @a="foo" @b="" />
```

## Versions

Seeing this behaviour both on `ember-template-lint@5.11.2` and latest (`ember-template-lint@6.0.0`)

**Comment by IgnaceMaes** [comment]
Seems like this was reported before: https://github.com/ember-template-lint/ember-template-lint/issues/2692

And mentioned as fixed in https://github.com/ember-template-lint/ember-template-lint/pull/2701

Why am I still seeing this issue then? 🤔

**Comment by lifeart** [comment]
That's correct issue still exists, and likely related to https://github.com/ember-template-lint/ember-template-recast
